bundle/manifests/nsx-container-plugin-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,375 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+     [
+       {
+         "apiVersion": "operator.nsx.vmware.com/v1",
+         "kind": "NcpInstall",
+         "metadata": {
+           "name": "ncp-install"
+         },
+         "spec": {
+           "ncpReplicas": 1
+         }
+       } 
+     ]
+    capabilities: Seamless Upgrades
+    categories: Networking, Security
+    certified: "True"
+    containerImage: vmware/nsx-container-plugin-operator:0.1.0
+    description: An operator which provides NSX as default network for an Openshift
+      cluster. Simplifies the process of installing and upgrading the NSX Container
+      plugin (NCP) components running in an Openshift cluster. The operator also allows
+      for detailed monitoring of NCP components and reacts to configuration changes.
+      This is a cluster operator and must be installed upon cluster creation; for
+      more information please refer to install-time instructions.
+    marketplace.openshift.io/action-text: install-time Instructions
+    marketplace.openshift.io/remote-workflow: https://docs.vmware.com/en/VMware-NSX-T-Data-Center/3.0/ncp-openshift/GUID-303605D3-6447-45FE-8FE1-3CAF0F7FC578.html
```

Comment:
There is more recent doc here: https://docs.vmware.com/en/VMware-NSX-T-Data-Center/3.1/ncp-openshift/GUID-1D75FE92-051C-4E30-8903-AF832E854AA7.html

Reply:
> There is more recent doc here: https://docs.vmware.com/en/VMware-NSX-T-Data-Center/3.1/ncp-openshift/GUID-1D75FE92-051C-4E30-8903-AF832E854AA7.html

Thanks, fixed.